about.html

```diff
@@ -0,0 +1,150 @@
+---
+layout: default
+description: We build open-source, human-centered, secure, agile solutions to support the delivery of government services that increase equity of opportunity.
+---
+
+<section class="row d-flex align-items-center h-100vh-minus-nav position-relative">
+    <div class="container">
+        <div class="offset-lg-1 col-lg-7 py-5 my-5 py-md-0 my-md-0">
+            <span class="pill">About Us</span>
+            <h1 class="mt-4 mb-3">We make government services accessible for all</h1>
+            <p class="mb-4 text-white">At Compiler we’re dedicated to building open-source, human-centered, secure, solutions to support the delivery of government services that increase equity of opportunity.</p>
+            <a class="btn btn-primary mb-4 mb-md-0" target="_blank" href="/capabilities">Read our Capabilities Statement</a>
+        </div>
+        <span class="animate-bus">
+            <img alt="Illustration of a bus" src="/assets/illustrations/bus.svg">
```

Comment:
I love this little guy! In a future iteration, it would be nice if we could force it fully above the fold, so the entire bus is visible when someone lands on this page:

![image](https://github.com/compilerla/compiler.la/assets/1783439/4609964c-7fae-4a8f-bf80-bf0bd794f560)

Reply:
It should be above the fold fully:

<img width="259" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/837c39ca-8f6d-4607-a187-8eafb6efbc3f">

Reply:
But we can't have it 100% always above the fold, *IF* the height is not tall enough to fix all the text -- or else, the bus will run into the button. So that's why there's the a pretty tall `min-height` there for mobile screens, to avoid this situation:
<img width="1512" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/20463f30-af15-41cc-899a-065b950a29f7">
<img width="1512" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/0631ef4e-7c1e-4c7c-ab55-503f9024e08a">

Reply:
Right, I totally get that. My screenshot is just from a laptop screen, is there a way to shorten that min-height without impacting mobile? It is definitely above the fold on a normal desktop monitor.

Reply:
Try this: https://github.com/compilerla/compiler.la/pull/119/commits/8fa16ff205ebc85b17e555e745490ae5232ba013

Try this at your regular browser text size level, and then try boosting up to 200% font size -- it shouldn't cause the bus/button to clash.

Reply:
Looks great!
![image](https://github.com/compilerla/compiler.la/assets/1783439/109b6216-01c7-42c1-bf8c-7504862dcc78)